src/compiler/crystal/syntax/lexer.cr

```diff
@@ -1966,7 +1966,7 @@ module Crystal
           elsif !delimiter_state && whitespace && (keyword = lookahead { check_macro_opening_keyword(beginning_of_line) })
             char = current_char
 
-            nest += 1 unless keyword == :abstract_def
+            nest += 1 unless keyword == {Keyword::ABSTRACT, Keyword::DEF}
```

Comment:
Considering that this seems to be the only use of `check_macro_opening_keyword`, wouldn't it be sufficient to return `true`/`false` depending on whether it found `abstract def`? 🤔

Reply:
The truthiness is also used, there'll need to be at least 3 distinct return values

Reply:
There might be a way